include/kq.h

```diff
@@ -116,6 +116,49 @@ class KMap
     std::vector<int> zone_array;
 };
 
+class KInventory
+{
+  public:
+    using Items = std::vector<s_inventory>;
+    /// Get number of items in the inventory
+    Items::size_type size() const
+    {
+        return inv.size();
+    }
+    // Manipulate functions
+    /*! \brief Add to the inventory
+     * \param item the I_XXX id to add
+     * \param quantity the quantity to add
+     */
+    void add(int type, int quantity = 1);
+    /*! \brief Remove from to the inventory
+     * \param item the I_XXX id to remove
+     * \param quantity the quantity to remove
+     * \returns true if there was sufficient to remove the quantity requested
```

Comment:
It appears that, if the quantity requested is more than the quantity owned, it will remove as many as possible and return false?

So I own 5 and `remove(I_XXX, 10)` will remove 5 of them and return false. Is that correct?